src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -1048,42 +1076,47 @@ export function getAlertContent(alert) {
               If you need help with understanding your Statement of the Case or completing the Form
               9, contact your VSO or representative.
             </p>
-            <p><a href="#">[LINK] Learn more about completing the Form 9</a>.</p>
           </div>
         ),
         displayType: 'take_action',
         type
       };
-    case ALERT_TYPES.scheduledHearing:
+    }
+    case ALERT_TYPES.scheduledHearing: {
+      const formattedDate = moment(alert.date, 'YYYY-MM-DD').format('MMMM DD, YYYY');
       return {
-        title: 'A hearing has been scheduled',
-        description: (
-          <p>Your [TYPE] hearing has been scheduled for [DATE] at [LOCATION].</p>
-        ),
+        title: `Your hearing is scheduled for ${formattedDate}`,
+        description: null,
         displayType: 'take_action',
         type
       };
-    case ALERT_TYPES.hearingNoShow:
+    }
+    case ALERT_TYPES.hearingNoShow: {
+      const formattedDate = moment(alert.date, 'YYYY-MM-DD').format('MMMM DD, YYYY');
```

Comment:
Where are you getting these detail fields from to know this should be `alert.date`? Is there a list somewhere?

Reply:
Yep, from the caseflow api progress ticket:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/7023